crates/edr_provider/src/data.rs

```diff
@@ -3732,12 +3732,6 @@ mod tests {
             chain_id: 1,
             url: get_alchemy_url(),
         },
-        // This block has both EIP-2930 and EIP-1559 transactions
```

Comment:
We should migrate this test to sepolia, as testing of a block with both an Eip-2930 and Eip-1559 transaction is what's really tested here

Reply:
Done. Thanks @fvictorio for sending the tx!